src/renderers/testing/ReactTestRendererFiberEntry.js

```diff
@@ -611,6 +605,21 @@ var ReactTestRendererFiber = {
         return TestRenderer.getPublicRootInstance(root);
       },
     };
+
+    Object.defineProperties(entry, {
+      root: {
+        get: function() {
+          if (root === null || root.current.child === null) {
+            throw new Error("Can't access .root on unmounted test renderer");
+          }
+          return wrapFiber(root.current.child);
+        },
+        enumerable: true,
+        configurable: true,
```

Comment:
`enumerable` perhaps but I don't think it should be `configurable`. (Both of these default to `false` by the way, which I think would also be reasonable in this case.) 😄 Not a big deal though just sharing my thoughts